History.txt

```diff
@@ -0,0 +1,6 @@
+=== 1.0.0 / 2018-05-09
```

Comment:
I'd change this to `CHANGELOG.md` in order for:

1. Make it clear we want markdown here, and not just plain text
2. be consistent in the naming of these files in all uppercase

Reply:
I tried to do this, but [hoe](http://yukimotopress.github.io/gem-tasks) didn't allow History.txt to disappear. And hoe can help create many useful rake tasks, so I think we can make a compromise here.